apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-data-template.md

```diff
@@ -0,0 +1,248 @@
+---
+title: 'Tutorial: How to Create a Component Data Template in XM Cloud'
+description: 'Learn about defining the content model that you can use for data sources in the XM Cloud Component builder.'
+openGraphImage: 'https://sitecorecontenthub.stylelabs.cloud/api/public/content/f22899c6baf54eb2bfcf23eb033e9107?v=0e490ed4'
+pageType: 'tutorial'
+menuOrder: 4
+---
+
+<Introduction title="What You are Going to Learn">
+
+In this tutorial, you will go through the steps to create a new content model that can be used for your Text Teaser component. You will learn:
+
+- How to create a headless module in XM Cloud Content Editor
+- How create new data templates for the Text Teaser component in XM Cloud Content Editor
```

Comment:
How **to** create

Reply:
Oops! Fixing that typo in the next commit.